src/screens/HelpScreen.tsx

```diff
@@ -119,34 +145,34 @@ const styles = StyleSheet.create({
 });
 
 const data = [
+  // {
+  //   text: ACC_STRS.contactSubtitle,
+  //   style: styles.contactTitle,
+  // },
   {
-    text: AS.contactSubtitle,
-    style: styles.contactTitle,
-  },
-  {
-    text: AS.contactStreet,
+    text: ACC_STRS.contactStreet,
     style: styles.contactText,
   },
   {
-    text: `${AS.contactPostalCode} ${AS.contactCity}`,
-    style: styles.contactText,
-  },
-  {
-    text: AS.phone,
-    style: styles.contactTitle,
-  },
-  {
-    text: AS.contactPhone,
-    style: styles.contactText,
-  },
-  {
-    text: AS.email,
-    style: styles.contactTitle,
-  },
-  {
-    text: AS.contactEmail,
+    text: `${ACC_STRS.contactPostalCode} ${ACC_STRS.contactCity}`,
     style: styles.contactText,
   },
+  // {
```

Comment:
Zou dit gewoon helemaal weg laten als het toch uitgecomment is

Reply:
Mijn gedachte is dat het best wel handig is om nog even te behouden. 
Dit scherm  gaat (mocht het verder uitegewerkt worden aangezien het meer een nice to have is) later nog door veel development heen. Deze data (testdata?) staat al klaar. Dan kan je lekker snel aan de slag gaan. 
Commenten kan geen kwaad.